README.md

```diff
@@ -1,82 +1,71 @@
-# splits i/o
-splits i/o is a website similar to Pastebin or GitHub Gist, but for splits generated from speedruns rather than text or
+# Splits I/O
+Splits I/O is a website similar to Pastebin or GitHub Gist, but for splits generated from speedruns rather than text or
 code. It's written in Ruby on Rails.
 
-splits i/o currently supports splits from Urn, LiveSplit, SplitterZ, Time Split Tracker, and WSplit.
+Splits I/O currently supports splits from Urn, LiveSplit, SplitterZ, Time Split Tracker, and WSplit.
+
+## API
+For full API documentation, see the [API readme][api-docs].
+
+[api-docs]: ./docs/api.md
 
 ## Running locally
+Splits I/O runs on Docker, which makes it easy and consistent to set up and run on any machine despite any unusual
+dependencies. The one downside is that you must first install Docker!
 
 ### Requirements
-* Ruby 2.3.4
-* PostgreSQL (make sure you've run `createdb`)
+* [Docker][docker-download]
+
```

Comment:
Should docker-compose be added as well since you have to install that separately?

Reply:
Good point, didn't realize it didn't always come with Docker.

Reply:
At least for Ubuntu it has you install docker first, and then you pull down the docker-compose script